### PR TITLE
Material for MkDocs

### PR DESCRIPTION
**Pull request by jodygarnett**
Content converted using pandoc, this is a prototype ahead of boldens code sprint.

I am neutral on MkDocs, but appreciate markdown as it is used in so many other places. Material for MkDocs has so many extensions that we can get 90% of sphinx-build funcitonality. 

The only real weakness is including live examples which is done using line-number range (making it hard to fix documentation if a configuration file being documented changes).

```
pip3 install -r requirements.txt
mkdocs serve
open http://localhost:8000
```

See README.md in this pull request for alternate setup instructions, or consult referencess below:

* https://squidfunk.github.io/mkdocs-material/getting-started/
* https://squidfunk.github.io/mkdocs-material/reference/
* https://www.mkdocs.org/user-guide/writing-your-docs/

image

Compare:

* https://jodygarnett.github.io/iso19139.ca.HNAP/
* https://metadata101.github.io/iso19139.ca.HNAP/

### CODE REVIEW

**Comment by jodygarnett** [comment]
I think I have a live preview now at https://jodygarnett.github.io/iso19139.ca.HNAP/

File: .gitignore [hunk 1]
```diff
@@ -27,6 +27,11 @@ target/
 .*/
 out/
 
+
+# Python stuff
+venv
+site
+
 # Sphinx-build
```
Comment (ianwallen) [inline .gitignore:34]: If you are getting ride of Sphinx do we still need this?

File: pom.xml [hunk 2]
```diff
@@ -216,6 +216,7 @@
               </execution>
             </executions>
           </plugin>
+          <!--
```
Comment (ianwallen) [inline pom.xml]: If not required then shouldn't it be removed?

**Comment by jodygarnett** [comment]
I will remove sphinx-build when I have mkdocs working. I need to sort out translation … and it is being very difficult.

**Comment by jodygarnett** [comment]
This content has moved to core-geonetwork

**Pull request closed without merging**